src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1,160 +1,164 @@
-              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
-              <xsd:attribute ref="xml:space" />
-            </xsd:complexType>
-          </xsd:element>
-          <xsd:element name="resheader">
-            <xsd:complexType>
-              <xsd:sequence>
-                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
-              </xsd:sequence>
-              <xsd:attribute name="name" type="xsd:string" use="required" />
-            </xsd:complexType>
-          </xsd:element>
-        </xsd:choice>
-      </xsd:complexType>
-    </xsd:element>
-  </xsd:schema>
-  <resheader name="resmimetype">
-    <value>text/microsoft-resx</value>
-  </resheader>
-  <resheader name="version">
-    <value>2.0</value>
-  </resheader>
-  <resheader name="reader">
-    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </resheader>
-  <resheader name="writer">
-    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </resheader>
-  <data name="AppDisplayName" xml:space="preserve">
-    <value>Microsoft.PowerToys.Settings.UI</value>
-    <comment>Application display name</comment>
-  </data>
-  <data name="AppDescription" xml:space="preserve">
-    <value>Microsoft.PowerToys.Settings.UI</value>
-    <comment>Application description</comment>
-  </data>
-  <data name="Shell_Main.Content" xml:space="preserve">
-    <value>Main</value>
-    <comment>Navigation view item name for Main</comment>
-  </data>
-  <data name="Shell_General.Content" xml:space="preserve">
-    <value>General</value>
-    <comment>Navigation view item name for General</comment>
-  </data>
-  <data name="Shell_PowerLauncher.Content" xml:space="preserve">
-    <value>PowerLauncher</value>
-    <comment>Navigation view item name for PowerLauncher</comment>
-  </data>
-  <data name="Shell_PowerRename.Content" xml:space="preserve">
-    <value>PowerRename</value>
-    <comment>Navigation view item name for PowerRename</comment>
-  </data>
-  <data name="Shell_ShortcutGuide.Content" xml:space="preserve">
-    <value>Shortcut Guide</value>
-    <comment>Navigation view item name for Shortcut Guide</comment>
-  </data>
-  <data name="Shell_PowerPreview.Content" xml:space="preserve">
-    <value>File Explorer Preview</value>
-    <comment>Navigation view item name for File Explorer Preview</comment>
-  </data>
-  <data name="Shell_FancyZones.Content" xml:space="preserve">
-    <value>FancyZones</value>
-    <comment>Navigation view item name for FancyZones</comment>
-  </data>
-  <data name="Shell_ImageResizer.Content" xml:space="preserve">
-    <value>Image Resizer</value>
-    <comment>Navigation view item name for Image Resizer</comment>
-  </data>
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
can you do a quick diff on what changed on this file.

Reply:
Hey @laviusmotileng-ms  the diff is only limited to this pr changes now.